doc/api/functions.rst

```diff
@@ -0,0 +1,12 @@
+Top-level functions
+===================
+
+.. automodule:: sounddevice
```

Comment:
Please don't repeat the module docstring in this page.
I think it's sufficient to mention it in `api/index.rst`.

This should also solve one of the warnings.

Reply:
Done.